src/commands/config-schema.json

```diff
@@ -100,24 +100,44 @@
         "colors": {
             "type": "object",
             "description": "Mapping from jj formatter labels to color names",
+            "definitions": {
+                "colors": {
+                    "enum": [
+                        "black",
+                        "red",
+                        "green",
+                        "yellow",
+                        "blue",
+                        "magenta",
+                        "cyan",
+                        "white",
+                        "bright black",
+                        "bright red",
+                        "bright green",
+                        "bright yellow",
+                        "bright blue",
+                        "bright magenta",
+                        "bright cyan",
+                        "bright white"
+                    ]
+                }
+            },
             "additionalProperties": {
-                "enum": [
-                    "black",
-                    "red",
-                    "green",
-                    "yellow",
-                    "blue",
-                    "magenta",
-                    "cyan",
-                    "white",
-                    "bright black",
-                    "bright red",
-                    "bright green",
-                    "bright yellow",
-                    "bright blue",
-                    "bright magenta",
-                    "bright cyan",
-                    "bright white"
+                "oneOf": [
+                    { "$ref": "#/properties/colors/definitions/colors" },
+                    {
+                        "type": "object",
+                        "properties": {
+                            "fg": { "$ref": "#/properties/colors/definitions/colors" },
+                            "bg": { "$ref": "#/properties/colors/definitions/colors" },
+                            "bold": {
+                                "type": "boolean"
+                            },
+                            "underline": {
```

Comment:
I had previously asked on Discord if we should rename the attribute to "underline" because I kept making the mistake of calling it "underline" instead of "underlined". I'm kind of happy to see that you made the same mistake. Please update to "underlined" for now, but I'll probably send a PR to rename it to "underline" soon.

Reply:
Done in #1163. I haven't enabled auto-merge on that since we need to synchronize merging.

Reply:
Haha, glad to be in good company! Fixed.

Reply:
I just merged my PR, so please switch back to "underline" now. Sorry

Reply:
Lol alright, changed it back!